old-dyer-county-central-school.md

```diff
@@ -0,0 +1,14 @@
+---
+layout: center
+title: Old Dyer County Central School
+phone: 731-287-0775
+address: 1500 Hornbrook
```

Comment:
Can you add the full address here so Google will be able to geolocate it please?